pkg/network/opencontrail/network.go

```diff
@@ -123,6 +118,65 @@ func (m *NetworkManagerImpl) DeleteFloatingIpPool(network *types.VirtualNetwork,
 	return nil
 }
 
+// Subnets can only be deleted if there are no instance-ips or floating-ips associated
+// with the subnet.
+func (m *NetworkManagerImpl) updateSubnetConfig(network *types.VirtualNetwork, prefix string) {
```

Comment:
(question) I still don't understand why updateSubnetConfig trying to delete subnets which won't match prefix. 
Do we support only 1 prefix in a network?